content/en/docs/tasks/administer-cluster/kubeadm/adding-linux-nodes.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Adding Linux worker nodes
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+This page explains how to add Linux worker nodes to a kubeadm cluster.
+
+## {{% heading "prerequisites" %}}
+
+* Each joining worker node has installed the required components from
+[Installing kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/), such as,
+kubeadm, the kubelet and a {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}.
```

Comment:
This is the part to repeat (I've edited it, eg, to drop the 2 CPU requirement, which isn't true for worker nodes)

> - 2 GiB or more of RAM per server; any less will leave little room for your apps.
> - Full network connectivity between all machines in the cluster (public or private network is fine).
> - Unique hostname, MAC address, and product_uuid (unique for every node). See [here](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#verify-mac-address) for more details.
> - All the [required network ports](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#check-required-ports) allow connections from the control plane.

Reply:
but the user should navigate to the install kubeadm page if they haven't installed all the requirement components.

> (I've edited it, eg, to drop the 2 CPU requirement, which isn't true for worker nodes)

this can be clarified on the install page:
>> 2 CPUs or more for control plane machines.

> BTW I'm not sure if kubeadm actually requires that nodes have Ethernet.

the user can choose to deploy or local network if they like

Reply:
> the install kubeadm page

That's giving the prerequisites for a control plane, right? We could use tabs there to clarify the different requirements.
But yes, we can can explain that you need to have already installed `kubeadm`.

IMO a later PR should refactor it so we _don't_ send people there; the way you install `kubeadm` is not that similar between Windows and Linux.

Reply:
added:
> 2 CPUs or more for control plane machines.

to the install page in the latest commit.

Reply:
> That's giving the prerequisites for a control plane, right? 

no, that section was for machines in general CP or workers, so the `2CPU for CP` was needed.

Reply:
OK, this'll do!